mubench.reviewsite/templates/review.phtml

```diff
@@ -358,10 +360,29 @@ $columns_to_ignore = array(
                     <?php endif; ?>
                     </td></tr>
                 </table>
-                <input type="hidden" name="origin" value="<?= htmlspecialchars($origin_path); ?>">
                 <input type="hidden" name="misuse_id" value="<?= htmlspecialchars($misuse->id); ?>">
-                <input type="submit" value="Save Review" />
-                </form>
+                <div style="position:fixed;bottom:0;left:0;width:100%;background-color:#f1f1f1;z-index:1;padding:5px;">
+                    <?php
+                        $origin_path = $origin_path ? $origin_path : $pathFor('experiment.detector', array("experiment_id"=> $experiment->id, "detector_muid" => $detector->muid));
+                        $next_misuse_path = $next_misuse ? $pathFor('view', array('experiment_id' => $experiment->id, 'detector_muid' => $detector->muid, 'project_muid' => $next_misuse->getProject(), 'version_muid' => $next_misuse->getVersion(), 'misuse_muid' => $next_misuse->misuse_muid)) . "?origin=$origin_path" . ($experiment->id === 2 ? "&ex2_review_size=$ex2_review_size" : "") : "";
+                        $next_reviewable_misuse_path = $next_reviewable_misuse ? $pathFor('view', array('experiment_id' => $experiment->id, 'detector_muid' => $detector->muid, 'project_muid' => $next_reviewable_misuse->getProject(), 'version_muid' => $next_reviewable_misuse->getVersion(), 'misuse_muid' => $next_reviewable_misuse->misuse_muid)) . "?origin=$origin_path" . ($experiment->id === 2 ? "&ex2_review_size=$ex2_review_size" : "") : "";
+                        $previous_misuse_path = $previous_misuse ? $pathFor('view', array('experiment_id' => $experiment->id, 'detector_muid' => $detector->muid, 'project_muid' => $previous_misuse->getProject(), 'version_muid' => $previous_misuse->getVersion(), 'misuse_muid' => $previous_misuse->misuse_muid)) . "?origin=$origin_path" . ($experiment->id === 2 ? "&ex2_review_size=$ex2_review_size" : "") : "";
+                        ?>
+                    <a href="<?= $origin_path ?>" style="padding:5px;text-align: center;" class="button">return</a>
+                    <a href="<?= $previous_misuse_path ?>" style="padding:5px;text-align: center;" class="button <?= $previous_misuse_path ? "" :  "disabled" ?>"><</a>
```

Comment:
I understand that the placement of the buttons is specific to this navigation bar, but why do you add the padding here as well? Shouldn't this be part of the class? (the question applies to all elements in this navigation bar)